datamade/how-to
# Add templates for containerizing Python web applications

## Overview

This pull request templates the Dockerfile, docker-compose.yml, and tests/docker-compose.yml recommended in our documentation for containerization for local development.

Closes #41 

## Testing Instructions

* Select a Django web application.
* Follow the instructions in `docker/templates/README.md` to generate containerization artifacts for your application.
* Build and run your newly containerized application, and confirm that it runs as expected.
* View the rendered markdown for local development and the templates README, and confirm that links work and the formatting looks nice.
